docs/cli-usage.md

```diff
@@ -0,0 +1,144 @@
+# Using the SOCI CLI
+
+This document provides instructions on how to use the SOCI CLI. And a comprehensive list of all the cli commands in one place.
+## Global flags
+
+- ```--address``` , ```-a```: Address for containerd's GRPC server
+- ```--namespace```, ```-n```: Namespace to use with commands
+- ```--timeout```: Timeout for commands
+- ```--debug```: Enable debug output
+- ```--content-store```: Use a specific content store (soci or containerd)
+
+## CLI commands
+- [soci create](#soci-create)
+- [soci push](#soci-push)
+- [soci rebuild_db](#soci-rebuild_db)
+- [soci index](#soci-index)
+- [soci ztoc](#soci-ztoc)
+
+
+### soci create
+Creates SOCI index for an image
+
+Output of this command is SOCI layers and SOCI index stored in a local directory
+>SOCI layer is named as \<image-layer-digest>.soci.layer
+>
+>SOCI index is named as \<image-manifest-digest>.soci.index
+
+Usage: ```soci create [flags] <image_ref> ```
+
+Flags:
+
+ - ```--spanSizeFlag``` : Span size that soci index uses to segment layer data. Default is 4MiB 
+ - ```--minLayerSizeFlag``` : Minimum layer size to build zTOC for. Smaller layers won't have zTOC and not lazy pulled. Default is 10MiB
```

Comment:
Looks like accidentally grabbed the variable names instead of the flags themselves.

Reply:
These are the names of the flags, however I jsut realized that these are platform flags. Not sure if these flags need to be documented.

Definition in file : https://github.com/awslabs/soci-snapshotter/blob/7af8577ac627f991d561f94d476ea4529a02626a/cmd/soci/commands/create.go#L40

Reply:
For example, `spanSizeFlag` is a constant with value `span-size`. So the command line flag users will provide will be `--span-size`.

https://github.com/awslabs/soci-snapshotter/blob/7af8577ac627f991d561f94d476ea4529a02626a/cmd/soci/commands/create.go#L32

Reply:
My bad, fixed will fix this.